docker/application.properties

```diff
@@ -3,7 +3,7 @@
 eip.home=/eip-home
 
 # Camel endpoints that need to be notified of DB events
-db-event.destinations=direct:outbound-lims,direct:test_order-from-order-creator,direct:odoo-event-listener
+# db-event.destinations=direct:outbound-lims,direct:test_order-from-order-creator,direct:odoo-event-listener
```

Comment:
Are you sure about commenting this line?

Reply:
Yeah that line assumes all clients will have the same routes so it cause and issue as this routes are preferred to the ones in the distro.

Reply:
Our distro sets this https://github.com/ozone-his/ozone-config/blob/d77ddaa871f92a88b76261774da3ce7281e825ac/eip_config/config/application.properties#L33 so it will be fine. The ICRC on does the same as well.